api/frontend.ts

```diff
@@ -24,14 +24,13 @@ export function uiConfig (req, res) {
     const stream = fs.createReadStream(env.AUTH_CONFIG_FILE)
     stream.pipe(res)
   } else {
-    res.write(`
+    res.end(`
```

Comment:
Shouldn't that be `res.send`?

Reply:
`end` works, but I guess `send` does more useful things